ovos_padatious/intent_manager.py

```
@@ -48,6 +48,14 @@ def calc_intents(self, query: str, entity_manager) -> List[MatchData]:
             List[MatchData]: A list of matches sorted by confidence.
         """
         sent = tokenize(query)
+        if not self.objects:
+            return []
+        if len(self.objects) == 1:
+            try:
+                match = self.objects[0].match(sent, entity_manager).detokenize()
+                return [match]
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)